packages/relay-compiler/transforms/__tests__/__snapshots__/RelayApplyFragmentArgumentTransform-test.js.snap

```diff
@@ -91,6 +91,69 @@ fragment Profile_4CNNX6 on User {
 
 `;
 
+exports[`RelayApplyFragmentArgumentTransform matches expected output: literal-argument.graphql 1`] = `
+~~~~~~~~~~ INPUT ~~~~~~~~~~
+query TestQuery(
+  $id: ID!
+) {
+  node(id: $id) {
```

Comment:
Should we add a test case for `pictureSize` coming in as a global query variable as well?
```
query TestQuery(
  $id: ID!, pictureSize: Int!
) 
```

Reply:
The new test also has a global variable